resources/js/lang/en.js

```diff
@@ -1,12 +1,26 @@
 export default {
+  route: {
+    profile: 'Profile',
+    notifications: 'Notifications',
+    groups: 'My Groups',
+  },
   auth: {
     login: 'Login',
     signUp: 'Sign Up',
+    signOut: 'Sign Out',
     emailAddress: 'Email address',
     password: 'Password',
     rememberMe: 'Remember me',
     enterEmail: 'Enter email',
     enterPassword: 'Enter password',
     notHaveAccountYet: 'Not have account yet?',
   },
+  user: {
+    email: 'Email',
+    birthday: 'Birthday',
+    gender: 'Gender',
+    phone: 'Phone',
+    address: 'Address',
+    yourGroups: 'Your Groups',
+  },
```

Comment:
cho em hỏi là trong phần đa ngôn ngữ của VueX thì nên để file là `en.js` hay là `en.json` hơn ạ?

Reply:
@vietnc-1636 
theo hướng dẫn thì `.js` á Việt.
không biết có best-practice nào không ^^

Reply:
em nghĩ chắc tùy người đặt file là gì thôi ạ, vì `.js` hay `.json` đều được.
Em chưa rõ phần này nên em hỏi chị thôi ạ, em toàn để `.json` thôi :v